Assets/Scripts/GameManager.cs

```diff
@@ -7,21 +7,12 @@ public class GameManager : MonoBehaviour
 {
     public GameObject gameOverUI;
     public ScoreManager scoreManager;
-   private bool playerDead = false;
```

Comment:
I think this section should be under a different game state as this causes the item screen bug

Reply:
ItemSelect enum added